build/js/main.min.js

```diff
@@ -0,0 +1 @@
:2e3})}).error(function(e){l.create({className:"danger",content:e.message,dismissOnTimeout:!0,dismissOnClick:!0,timeout:2e3})})},n.logout=function(){o.removeToken(),s.$storage.$reset(),e.logout(function(){"home"===r.$current.name?r.reload():r.go("home"),l.create({className:"success",content:"successfully logged out",dismissOnTimeout:!0,dismissOnClick:!0,timeout:2e3})},function(){l.create({className:"danger",content:"failed to logout!",dismissOnTimeout:!0,dismissOnClick:!0,timeout:2e3})})},n.authenticate=function(e){o.authenticate(e).then(function(e){o.setToken(e.data.token),s.$storage.activeUser=e.data.user.username,s.$storage.userId=e.data.user._id,r.go("home"),s.$storage.loggedIn=!0,l.create({className:"success",content:"You are signed in",dismissOnTimeout:!0,dismissOnClick:!0,timeout:2e3})})["catch"](function(e){l.create({className:"danger",content:e.data.message,dismissOnTimeout:!0,dismissOnClick:!0,timeout:2e3})})},n.isAuthenticated=function(){o.isAuthenticated()}}]),angular.module("hairvenApp").value("baseUrl","http://localhost:3030");var app=angular.module("hairvenApp");app.factory("HairstyleService",["$http","baseUrl","$localStorage","Upload",function(e,t,s,a){var n={getAllHairstyles:function(){return e.get(t+"/api/hairstyles/")},getSalonHairstyles:function(s){return e.get(t+"/api/salons/"+s+"/hairstyles")},getOneHairstyle:function(s){return e.get(t+"/api/hairstyles/"+s)},addHairstyle:function(e,s){return a.upload({url:t+"/api/hairstyles",data:{hairPhoto:e,name:s.name,description:s.description,salon:s.salon},method:"POST"})},updateHairstyle:function(s,a){return e.put(t+"/api/hairstyles/"+s,a)},deleteHairstyle:function(s){return e["delete"](t+"/api/hairstyles/"+s)}};return n}]);var app=angular.module("hairvenApp");app.factory("SalonService",["$http","baseUrl","$localStorage",function(e,t,s){var a={getSalon:function(s){return e.get(t+"/api/salons/"+s)},addSalon:function(s){return e.post(t+"/api/salons/",s)},updateSalon:function(s,a){return e.put(t+"/api/salons/"+s,a)},deleteSalon:function(s){return e["delete"](t+"/api/salons/"+s)}};return a}]);var app=angular.module("hairvenApp");app.factory("UserService",["$http","baseUrl","$localStorage",function(e,t,s){function a(e){var t=e.replace("-","+").replace("_","/");switch(t.length%4){case 0:break;case 2:t+="==";break;case 3:t+="=";break;default:throw"Illegal base64url string!"}return window.atob(t)}function n(){var e=s.token,t={};if("undefined"!=typeof e){var n=e.split(".")[1];t=JSON.parse(a(n))}return t}function r(e){angular.extend(i.currentUser,e)}var i=(n(),{register:function(s){return e.post(t+"/api/signup",s)},login:function(s){return e.post(t+"/api/login",s)},facebookLogin:function(){e.get(t+"/api/auth/facebook/")},twitterLogin:function(){e.get(t+"/api/auth/twitter/")},logout:function(e){r({}),e()},currentUser:function(){n()},updateUser:function(s,a,n){e.put(t+"/api/users/"+s).success(a).error(n)},deleteUser:function(s,a,n){e["delete"](t+"/api/users/"+s).success(a).error(n)}});return i}]);
```

Comment:
Line is too long.<br>Expected an assignment or function call and instead saw an expression.<br>Missing semicolon.<br>A leading decimal point can be confused with a dot: '.7'.<br>Too many errors. (100% scanned).